docs/source/schema-design/federated-schemas/entities/intro.mdx

```diff
@@ -1,24 +1,24 @@
 ---
 title: Introduction to Entities
-subtitle: Resolve federated types across multiple subgraphs
-description: Learn to define, contribute to, and reference object types that resolve their fields across multiple subgraphs in a federated GraphQL architecture.
+subtitle: Fundamental keyed object type of Apollo Federation
+description: Learn to define, contribute to, and reference entities, the fundamental object types of Apollo Federation that resolve their fields across one or more subgraphs.
 redirectFrom:
     - /federation/entities
 ---
 
-Federated schemas let multiple subgraphs collaboratively define and resolve fields for shared object types.
-This guide shows you how to define a shared object type called an _entity_.
+<Note>
 
 Before you get started, you may want to check out the [Introduction to Apollo Federation](/graphos/schema-design/federated-schemas/federation) for a conceptual overview.
```

Comment:
W/o an introductory sentence before the <Note>, I think this sentence would benefit w/ a little nod to what this page will cover.

```suggestion
Before getting started with entities, you may want to check out the [Introduction to Apollo Federation](/graphos/schema-design/federated-schemas/federation) for a conceptual overview.
```

Reply:
another future todo: add and use a Prerequisites callout